unit/java_bytecode/java_bytecode_parser/lambda_examples/readme.md

```diff
@@ -0,0 +1,19 @@
+Each sub directory inside this one contains class files which have been compiled with specific java compilers. The proceedure taken to produce these files is as follows -
+ 1. Install the java compilers of interest. Skip this step if you already have them installed.
+ 2. Use `sudo update-alternatives --config javac` to configure your system to use the compiler of interest.
+ 3. Use `javac -version` to confirm the desired compilier has been correctly configured. The version information for each compiler is as follows -
+    * openJdk 8     - 1.8.0_151
+    * oracle java 8 - 1.8.0_161
+    * oracle java 9 - 9.0.4
+ 4. Compile the `.java` files using `javac -g ./*.java`. The `-g` flag switches on debugging output, which we require.
+ 5. The `.class` files should have been written to the current directory. They can now be moved into the appropriate sub directory. For example - `mkdir my_compiler_classes && mv ./*.class ./my_compiler_classes`
+ 6. If this is a new compiler, add it to the test, by adding the compilier to the list in the `run_test_with_compilers` function in `java_bytecode_parse_lambda_method_table.cpp`.
+ 7. Repeat steps 2 through 6 for each additional compiler.
+
+The proceedure for compiling with eclipse is different. Installing the java eclipse ide does not setup a version of javac command. So to produce the class files for eclipse, the proceedure is as follows -
```

Comment:
procedure

Reply:
You should be able to run the Eclipse compiler as explained here: https://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.jdt.doc.user%2Ftasks%2Ftask-using_batch_compiler.htm